egs/librispeech/ASR/tdnn_lstm_ctc/train.py

```diff
@@ -260,14 +264,79 @@ def save_checkpoint(
         best_valid_filename = params.exp_dir / "best-valid-loss.pt"
         copyfile(src=filename, dst=best_valid_filename)
 
+class LossRecord(collections.defaultdict):
```

Comment:
Could you move it to `icefall/utils.py` so that it can be reused by other `xxx/train.py`?

Reply:
Also, could you ensure that GitHub actions get passed, i.e., fix style issues?

Reply:
> Could you move it to `icefall/utils.py` so that it can be reused by other `xxx/train.py`?

OK, I will do it. In fact, I also consider this method.

Reply:
> Also, could you ensure that GitHub actions get passed, i.e., fix style issues?

Let me see.